src/Orleans/Configuration/GlobalConfiguration.cs

```diff
@@ -593,6 +599,8 @@ internal GlobalConfiguration()
 
             ProviderConfigurations = new Dictionary<string, ProviderCategoryConfiguration>();
             GrainServiceConfigurations = new GrainServiceConfigurations();
+            DefaultVersionCompatibilityStrategy = BackwardCompatible.Singleton;
+			DefaultVersionPlacementStrategy = MinimumVersionPlacement.Singleton;
```

Comment:
@jdom maybe we should check in a `.editorconfig` file or something. I'm surprised VS didn't automatically detect the norm, though.